php/Dockerfile

```diff
@@ -68,8 +68,9 @@ RUN mkdir -p /var/www && \
     chmod +x /usr/local/bin/docker-entrypoint && \
     chmod +x /usr/local/bin/init-drupal
 
+ARG CMS_VERSION=8.x-1.x
```

Comment:
CONTENTA_VERSION?

Reply:
Ya, that's the contenta_version. Obviously, the base here. I added the CMS_VERSION as this is how it was identified somewhere in the contenta build. So, just took a cue off them.